runtime/catalog-schema-registry/src/test/java/io/aklivity/zilla/runtime/catalog/schema/registry/internal/config/SchemaRegistryOptionsConfigAdapterTest.java

```diff
@@ -66,11 +79,30 @@ public void shouldWriteOptions()
             .url("http://localhost:8081")
             .context("default")
             .maxAge(Duration.ofSeconds(300))
+            .keys(List.of("client1"))
+            .trust(List.of("serverca"))
+            .authorization("Basic dXNlcjpzZWNyZXQ=")
             .build();
 
         String text = jsonb.toJson(catalog);
 
+        String expected =
+            "{" +
+                "\"url\":\"http://localhost:8081\"," +
+                "\"context\":\"default\"," +
+                "\"max-age\":300," +
+                "\"keys\":[\"client1\"]," +
+                "\"trust\":[\"serverca\"]," +
+                "\"credentials\":" +
+                "{" +
+                    "\"headers\":" +
+                    "{" +
+                        "\"authorization\":\"Basic dXNlcjpzZWNyZXQ=\"" +
+                    "}" +
+                "}" +
+            "}";
```

Comment:
Use `"""` multiline string syntax instead, and if we need no whitespace, then `.replaceAll(...)` on the end to remove all whitespace before assignment to `expected`.